specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```diff
@@ -0,0 +1,1443 @@
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserAssignedIdentity"
+      }
+    },
+    "UserAssignedIdentity": {
+      "description": "User Assigned Identity",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of the user assigned identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of the user assigned identity."
+        },
+        "clientId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The clientId(aka appId) of the user assigned identity."
+        }
+      }
+    },
+    "PaginatedInferenceEndpointsList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning Inference Endpoint objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array of Machine Learning Inference Endpoint objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/InferenceEndpointResource"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "PaginatedInferenceEndpointDeploymentsList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning Inference Deployment objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array of Machine Learning Inference Deployment  objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/InferenceEndpointDeploymentResource"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "SystemData": {
+      "description": "Specifies system data of the resource.",
+      "properties": {
+        "createdBy": {
+          "type": "string"
+        },
+        "createdByType": {
+          "type": "string",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "UserTypes",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string"
+        },
+        "lastModifiedBy": {
+          "type": "string"
+        },
```

Comment:
I believe these two should be read-only and should have a format of data-time or something?